docs/ibexa_products/ibexa_commerce.md

```diff
@@ -0,0 +1,96 @@
+
+[[= product_name_com =]] is based on [Symfony]([[= symfony_doc =]]). With a help of documentation and trainings, any developer familiar with Symfony or simply PHP may learn how to use available extension points and extend the platform.
+
+Version control systems and environment variables allow you to deploy your extensions and settings on several environments, such as [Ibexa Cloud](ibexa_cloud_guide.md).
+
+[[= product_name_com =]] is built on [[[= product_name_headless =]]](headless.md) and includes all bundles, APIs, and [features that comes with Headless edition](headless.md#core-features) as well as [Experience features](experience.md#core-features).
+
+## Capabilities and benefits
+
+[[= product_name_com =]] includes the greatest capabilities to accelerate your digital transformation: Personalized workflows, corporate account management, customized catalogs, and more.
+
+### Core features
+
+[[= product_name_com =]] includes all the features you need to launch your online store and reduce the time it takes to go live.
+
+#### Order management
+
+With the advanced [Order management]([[= user_doc =]]/commerce/order_management/order_management/) tools, you can easily manage your customized purchase process.
+The Order management module allows users to review order details and updates, track order completion status, and cancel orders that are created when store customers purchase products, for a smooth purchasing process.
+You can also assign and maintain rights so that only authorized users and make modifications.
+When searching for orders you can use filters, so you save time and reduce effort.
+
+![Order management](order_management.png)
+
+#### Merchandizing
+
+Merchandizing assists in keeping the consistency of the brand and providing customized product recommendations with captivating visuals and powerful search features. 
+With this feature you can engage your customers with eye-catching graphics and information
+The search engine makes it easy to find what you're looking for by providing quick and easy access to the product catalogs.
+You can also use recommendations and campaigns to increase average order values and income.
+
+#### eCommerce
+
+With the [eCommerce feature]([[= user_doc =]]/personalization/use_cases/#ecommerce) you can transform your online stores and give your consumers great buying experiences, from initial contact to post-purchase support.
+No matter how complicated your product or sales process are, you can present your offer in an approachable way.
+Creating engaging and personalized shopping experiences with targeted offers and recommendations, helps you to boost sales.
+Within eCommerce, product recommendations can assist users in finding the exact item that meets their needs.
+Recommendations can be used to propose related, alternative, or complimentary products to users who are unsure what to buy.
+
+![eCommerce](ecommerce.png)
+
+### Use cases
```

Comment:
I think all of these below (except for "eCommerce") are just features. For example: "Relevant faceted search" is not an use case, it's a feature you can use to achieve the usecase ("display products to the user").

I'd focus on two "high level" use cases:
- B2C: I want to sell my products directly to the Customers - how can Ibexa Commerce help me? With Site Factory, Page Builder, PIM, ..., faceted search, ..., connect, Personalization, (lots of options here!)
- B2B: I want to sell my products to other Companies -> how can Ibexa Commerce help me? Almost the same as above, but expanded additionaly with Customer Portals, Customer Groups - which are tailored at B2B use cases and building the long-term relations with the customers(you don't have a Sales Rep in B2C, but you do in B2B).

I really like the use cases we have in https://ez-systems-developer-documentation--2460.com.readthedocs.build/en/2460/ibexa_products/headless/#use-cases for Headless- it describes the real-life situations when someone might consider using Headless (and it even mentions when to use Ibexa Commerce - to finally become a store and sell online - so I'd expand on that).

Reply:
Agree! Let's put Merchandising , consumer-facing web store and B2B the three use cases.